netron.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/lutzroeder/Netron",
+    "description": "Visualizer for deep learning and machine learning models.",
+    "version": "2.8.1",
+    "license": "MIT",
+    "url": "https://github.com/lutzroeder/netron/releases/download/v2.8.1/Netron-Setup-2.8.1.exe/#dl.7z",
+    "hash": "A4720555396204E66E1BD3326FDD123B962595197E50FFDA39871F02CE74676D",
+    "pre_install": "extract_7zip \"$dir\\`$PLUGINSDIR\\app-*.7z\" \"$dir\"",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",
+    "bin": "netron.exe",
+    "shortcuts": [
+        [
+            "netron.exe",
+            "Netron"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/lutzroeder/netron/releases/download/v$version/Netron-Setup-$version.exe/#dl.7z"
+    }
```

Comment:
Add hash extraction from `$baseurl/latest.yml`

Reply:
Thank you.